tests/e2e/create/testdata/moduleconfig/valid/local-manifest.yaml

```diff
@@ -0,0 +1,16 @@
+# Sample for the tests!
```

Comment:
please move to a separate folder holding manifests, similar to sec-scanners-config